static/js/balloon.js

```diff
@@ -151,6 +153,54 @@ function updateSummaryDisplay(){
         $("#summary_table").tabulator("redraw", true);
     }
 }
+function updateSummaryDisplayImperial1(){
+    
+    // Update the 'Payload Summary' display.
+    var _summary_update = {id:1};
+    // See if there is any payload data.
+    if (balloon_positions.hasOwnProperty(balloon_currently_following) == true){
+        // There is balloon data!
+        var _latest_telem = balloon_positions[balloon_currently_following].latest_data;
+        
+        _summary_update.alt = (_latest_telem.position[2]*chase_config['m_to_ft']).toFixed(0) + "ft (" + (_latest_telem.max_alt*chase_config['m_to_ft']).toFixed(0) + "ft)";
```

Comment:
I don't see any need to refer out to config parameters for what are essentially constants. These values are not going to change, so may as well hardcode them. Same applies in numerous other places.

Reply:
Hardcoded there and numerous other places.